README.md

```diff
@@ -15,8 +15,8 @@ Fluent UI Apple contains native UIKit and AppKit controls aligned with [Microsof
 #### Requirements
 
 - iOS 14+ or macOS 10.15+
```

Comment:
Don't we require iOS 15 now too?

Reply:
Or did we not merge that yet?

Reply:
We haven't dropped support for 14 quite yet, I don't think.